v2/internal/pkg/operator/catalog_handler.go

```diff
@@ -418,28 +442,29 @@ func isPreReleaseOfFilteredVersion(version string, chEntryName string, filteredV
 	return false
 }
 
-func handleRelatedImages(bundle declcfg.Bundle, operatorName string, copyImageSchemaMap *v2alpha1.CopyImageSchemaMap) []v2alpha1.RelatedImage {
+func handleRelatedImages(bundle declcfg.Bundle, operatorName string, copyImageSchemaMap *v2alpha1.CopyImageSchemaMap) ([]v2alpha1.RelatedImage, error) {
 	var relatedImages []v2alpha1.RelatedImage
 
 	for _, ri := range bundle.RelatedImages {
 		if strings.Contains(ri.Image, "oci://") {
-			internalLog.Warn("%s 'oci' is not supported in operator catalogs : SKIPPING", ri.Image)
-			continue
+			msg := fmt.Sprintf("invalid image: %s 'oci' is not supported in operator catalogs", ri.Image)
+			return relatedImages, errors.New(msg)
 		}
-		relateImage := v2alpha1.RelatedImage{}
+		relatedImage := v2alpha1.RelatedImage{}
 		if ri.Image == bundle.Image {
-			relateImage.Name = ri.Name
-			relateImage.Image = ri.Image
-			relateImage.Type = v2alpha1.TypeOperatorBundle
+			relatedImage.Name = ri.Name
+			relatedImage.Image = ri.Image
+			relatedImage.Type = v2alpha1.TypeOperatorBundle
 		} else {
-			relateImage.Name = ri.Name
-			relateImage.Image = ri.Image
-			relateImage.Type = v2alpha1.TypeOperatorRelatedImage
+			relatedImage.Name = ri.Name
+			relatedImage.Image = ri.Image
+			relatedImage.Type = v2alpha1.TypeOperatorRelatedImage
 		}
 
 		imgSpec, err := image.ParseRef(ri.Image)
 		if err != nil {
-			internalLog.Warn("error parsing image %s : %v", ri.Image, err)
+			msg := fmt.Sprintf("error parsing image %s: %s", ri.Image, err.Error())
+			return relatedImages, errors.New(msg)
```

Comment:
```suggestion
			return relatedImages, fmt.Errorf("error parsing image %s: %w", ri.Image, err)
```
So we still have access to the internal wrapped error in case we need.